metaverse_components/pet.js

```diff
@@ -9,8 +11,47 @@ const localQuaternion3 = new THREE.Quaternion();
 const localEuler = new THREE.Euler();
 const localMatrix = new THREE.Matrix4();
 
+const downQuaternion = new THREE.Quaternion(-0.7071067811865476, 0, 0, 0.7071067811865476);
+
+const lastDirection = new THREE.Vector3();
+
+const HEIGHT_FOLLOW_LIMIT_Y = 0.9;
+const PET_WIDTH = 1.04;

```

Comment:
This constant needs to be declared or computed on the pet metaversefile components. It should not be hardcoded.

Reply:
agree ❤